plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/WriterSchemaResolver.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kafka.schema;
+
+public interface WriterSchemaResolver
```

Comment:
Why do you need such empty interface? Why not to use `ContentSchemaReader`?

Reply:
For the file based table description supplier a different implementation is required for the content schema reader and the file based writer schema resolver. Will look into how to unify them.

Reply:
Update: I got rid of the WriterSchemaResolver, added more tests. Lmk when you have some time to take a look.